### ISSUE TITLE
Crash with ic2classic

### ISSUE THREAD

**Issue by DbpGaming** [FrogCraft-Rebirth/FrogCraft-Rebirth#54]
https://pastebin.com/Tcmp8krk - crash report
don't know if you plan to support ic2classic but just in case

**Comment by 3TUSK** [comment]
IC2Classic support is not on my TODO list sadly.
However - nevertheless I doubt if this is indeed an issue on my side.

**Comment by 3TUSK** [comment]
One thing you may not know is that IC2 Experimental has a built-in "profile selection" functionality now. It allows you to switch back to the "classic profile" which (last time I checked) is basically bringing back the original IC2-lf contents.  
In IC2Exp's `IC2.ini`, find:
```
[profiles]
; Pick a profile to use, default options provided by IC2 are Experimental and Classic
selected = Experimental
```
Change `Experimental` to `Classic`, save config, reboot game and see the profit.

**Comment by DbpGaming** [comment]
yeah i know about classic mode, it just that doesnt work with ic2classic addons, i actually prefer exp but because of gregtech classic and some other things use classic, from the little i looked it seems like it cant get water cells, but i never seen a crash report line reference be right so i dont know if thats correct

**Comment by DbpGaming** [comment]
i have a feeling its due to ic2c not having fluid cells be fluid containers but be items

**Comment by 3TUSK** [comment]
Honestly speaking, that's exactly my speculation.
